src/components/Layout/Layout.js

```diff
@@ -29,28 +29,6 @@ class Layout extends React.Component {
 
   static contextType = ReactReduxContext;
 
-  componentDidMount = () => {
```

Comment:
When I do a reload of the page https://deploy-preview-173--master-fiori-exp.netlify.com/search/?q=button the search bar is not visible and the search value is empty